api/schema.yaml

```diff
@@ -21,6 +21,23 @@ components:
         configuration:
           type: object
           description: Custom configuration associated with the given treatment
+    SelectedTreatmentMetadata:
```

Comment:
Since the parent schema is already prefixed with `SelectedTreatment`, should we keep this concise with `Metadata`?

Reply:
That might not be advisable, since the schema definitions are shared. The field inside `SelectedTreatment` is just called `Metadata` though.